rs/crypto/internal/crypto_lib/threshold_sig/bls12_381/src/ni_dkg/fs_ni_dkg/nizk_sharing.rs

```diff
@@ -313,16 +313,29 @@ pub fn verify_sharing(
         // LHS = product [C_i ^ x^i | i <- [1..n]]^x' * Y
         // RHS = product [y_i ^ x^i | i <- 1..n]^z_r * g_1^z_alpha
```

Comment:
As we are rearranging the verification equations to do a single multiexp, we should update this doc so that it matches lhs and rhs below.